### PR TITLE
Make minor grammar corrections/updates to async/async-await

### PR DESCRIPTION
**Pull request by jchue**
Split out and revised changes originally in #1612

### CODE REVIEW

File: 1-js/11-async/08-async-await/article.md [hunk 1]
```diff
@@ -286,22 +284,22 @@ let results = await Promise.all([
...
 
 ````
 
 ## Summary
 
 The `async` keyword before a function has two effects:
 
-1. Makes it always return a promise.
-2. Allows to use `await` in it.
+1. Makes it always return a promise
```
Comment (paroche) [inline 1-js/11-async/08-async-await/article.md]: Wording change OK, is an improvement.

But removing periods from bullet-points in summary sections is not, from what I've noticed, standard for this document.

File: 1-js/11-async/08-async-await/article.md [hunk 2]
```diff
@@ -286,22 +284,22 @@ let results = await Promise.all([
...
+2. Allows `await` to be used in it
 
 The `await` keyword before a promise makes JavaScript wait until that promise settles, and then:
 
-1. If it's an error, the exception is generated, same as if `throw error` were called at that very place.
+1. If it's an error, the exception is generated — same as if `throw error` was called at that very place.
```
Comment (paroche) [inline 1-js/11-async/08-async-await/article.md]: M-dash slightly better, could go either way on were/was in this context.

File: 1-js/11-async/08-async-await/article.md [hunk 3]
```diff
@@ -286,22 +284,22 @@ let results = await Promise.all([
...
 2. Otherwise, it returns the result.
 
-Together they provide a great framework to write asynchronous code that is easy both to read and write.
+Together they provide a great framework to write asynchronous code that is easy to both read and write.
 
-With `async/await` we rarely need to write `promise.then/catch`, but we still shouldn't forget that they are based on promises, because sometimes (e.g. in the outermost scope) we have to use these methods. Also `Promise.all` is a nice thing to wait for many tasks simultaneously.
+With `async/await` we rarely need to write `promise.then/catch`, but we still shouldn't forget that they are based on promises, because sometimes (e.g. in the outermost scope) we have to use these methods. Also `Promise.all` is nice when waiting for many tasks simultaneously.
```
Comment (paroche) [inline 1-js/11-async/08-async-await/article.md]: or "nice when we are waiting for".

File: 1-js/11-async/08-async-await/article.md [hunk 4]
```diff
@@ -67,7 +67,7 @@ f();
 
 The function execution "pauses" at the line `(*)` and resumes when the promise settles, with `result` becoming its result. So the code above shows "done!" in one second.
 
-Let's emphasize: `await` literally makes JavaScript wait until the promise settles, and then go on with the result. That doesn't cost any CPU resources, because the engine can do other jobs meanwhile: execute other scripts, handle events etc.
+Let's emphasize: `await` literally makes JavaScript wait until the promise settles, and then go on with the result. That doesn't cost any CPU resources, because the engine can do other jobs in the meantime: execute other scripts, handle events, etc.
```
[the remark this answers is not in the document]
Comment (jchue) [reply 1-js/11-async/08-async-await/article.md]: So the change is OK?

File: 1-js/11-async/08-async-await/article.md (hunk 1 above)
Comment (jchue) [reply 1-js/11-async/08-async-await/article.md]: Good point

File: 1-js/11-async/08-async-await/article.md (hunk 2 above)
Comment (jchue) [reply 1-js/11-async/08-async-await/article.md]: I've retained the "were".

**Comment by jchue** [comment]
Updated

File: 1-js/11-async/08-async-await/article.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (paroche) [reply 1-js/11-async/08-async-await/article.md]: Yes.

**Pull request merged**